### DISCUSSION TITLE
Use isotope peak/feature scanner to find iron-containing compounds

### DISCUSSION THREAD

**Discussion by JethroHemmann** [mzmine/mzmine#2258] [Q&A]
Hi!

I would like to use the isotope peak scanner (from Feature list methods -> Isotopes -> Isotope feature scanner) to identify Fe-bound metabolites which show the typical Fe isotope pattern. 

Below is an example of such an isotope pattern from a test data set:
![grafik](https://github.com/user-attachments/assets/8214ddaf-2122-4e92-a200-4cdafadbb92e)

I tried to use the isotope peak scanner with the following settings on a feature list where all isotope peaks of the pattern are present as separate features:

![grafik](https://github.com/user-attachments/assets/d732f81d-a048-4ecd-9229-18c6dc844416)

However, the task just results in "Element not found" on the test data set. 

Is there anything else that needs to be considered to use this mzmine feature?

**Comment by SteffenHeu** [comment]
Hi, 
the merge width is quite small and the rating is quite high. try 0.01 and 0.5 respectively and see if it annotates something

**Reply by JethroHemmann** [reply]
Hi Steffen, 

Thanks so much for your answer! Setting the merge width to 0.01 already helped a lot and we are now detecting the pattern shown above! 

However, for some other iron-bound compounds, the pattern (which looks very similar to the one shown above) is still not recognized (all the individual isotope peaks are present as features). We played with all the different parameters, but were not successful. 

We would greatly appreciate some more guidance. Would there be a possibility to share an MZmine project with you, such that you could have a look?
